guides/hack/05-functions/05-format-strings.md

```diff
@@ -36,3 +35,19 @@ function use_it(): void {
 `HH\FormatString<PlainSprintf>` will check that you've used the right
 number of arguments. `HH\FormatString<Str\SprintfFormat>` will also
 check that arguments match the type in the format string.
+
+## Format Specifiers
+See `PlainSprintf` for in-depth information on all format specifiers. 
+
+| Specifier | Expected Input | Expected Output                                                                                                     |
+|-----------|----------------|---------------------------------------------------------------------------------------------------------------------|
+| b         | int            | Binary (`63` => `111111`)                                                                                           |
+| c         | int            | ASCII character (`63` => `?`)                                                                                       |
+| d, u      | int            | Signed int (d: `-1` => `-1`) \| Unsigned int (u: `-1` => `18446744073709551615`)                                    |
+| e, E      | float          | Scientific Notation, as lowercase or uppercase (e: `0.34` => `3.4e` \| E: `0.34` => `3.4E`)                         |
+| f, F      | float          | Locale Floating Point Number (f) \| Non-Locale Floating Point Number (F)                                            |
+| g         | float          | Locale Floating Point Number or Scientific Notation (`3.141592653` => `3.14159` \| `3141592653` => `3.1415e+9`) (🥧) |
+| o         | int            | Octal (`63` => `77`)                                                                                                |
+| s         | string         | string                                                                                                              |
+| x, X      | int            | Hexadecimal, as lowercase or uppercase (x: `63` => `3f` \| X: `63` => `3F`)                                         |
```

Comment:
Likewise here, I'm not sure what `|` means. In C-like languages, including Hack, `||` means logical OR and `|` means bitwise OR.

Reply:
Sorry, it's my shorthand for or. Probably something that only makes sense in my head.